.github/workflows/on_push-main.yml

```diff
@@ -83,30 +83,38 @@ jobs:
         with:
           pattern: 'v*'
 
-      - name: Generate version name
-        id: version-name
-        uses: ./config/actions/version-name
-        with:
-          version-code: ${{ steps.version-code.outputs.version-code }}
-          version-name: ${{ steps.latest-tag.outputs.current-tag }}
+      - name: Automatically bump semantic version via cocogitto
+        id: versioning
+        run: |
+          git config user.name github-actions
```

Comment:
Should this stuff not live in an action.yml?

Reply:
I'm not sure that it matters if it's only being used the once

Reply:
ok, fair enough